bin/run_qc.py

```diff
@@ -177,5 +183,37 @@ def detect_bad_cycle_tile_in_run(dataset, args):
             print('Bad tiles are: ' + ', '.join([str(c) for c in bad_tiles[lane]]))
 
 
+def get_all_project_gvcfs(project_folder):
+        gvcfs = []
+        for path in os.walk(project_folder):
+            gvcf = [i for i in path[2] if i.endswith('.g.vcf.gz')]
+            if gvcf:
+                gvcfs.append(os.path.join(path[0], ''.join(gvcf)))
+        return gvcfs
+
+def peddy(dataset, args):
+    all_gvcfs = []
+    sample_ids = []
+    if args.samples:
+        cfg.merge(cfg['sample'])
+        for sample_id in args.samples:
+            sample_ids.append(sample_id)
+            s = get_documents(sample_id)[0]
```

Comment:
Might as well just use `get_document` here, since we're only taking the first element from `get_documents`, and the sample ID is a unique identifier anyway.